README.md

```diff
@@ -4,13 +4,39 @@ FFC payment services management
 
 ## Prerequisites
 
+- Azure App Registration Setup
 - Docker
 - Docker Compose
 
 Optional:
 - Kubernetes
 - Helm
 
+## Azure App Registration
+
+`ffc-pay-web` has been integrated into Azure App Registration using the msal-node npm packackage (https://www.npmjs.com/package/@azure/msal-node)
+
+For `ffc-pay-web` to authenticate it needs to be registered with Azure App Registration (https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+
+The following environment varibles need to be set:
+
+- AZURE_CLIENT_ID
+- AZURE_TENANT_ID
+- AZURE_CLIENT_SECRET
+- REDIRECT_URL
```

Comment:
Config specifies a default value for `REDIRECT_URL` so this one doesn't need to be set and can be dropped.

Though see my comment about the default values not working

Reply:
removed